### PR TITLE
fix: type markdown native event

### PR DESCRIPTION
**Pull request by nkdengineer**
### Details

Change type markdown native event to fix console error that shows up

### Related Issues

https://github.com/Expensify/App/issues/51466

### CODE REVIEW

**Comment by tomekzaw** [comment]
Hi @nkdengineer, thanks for the PR!

Let's wait for review from @Skalakid and@BartoszGrajdek.

**Review by Skalakid — APPROVED** [review]
LGTM

**Review by BartoszGrajdek — APPROVED** [review]
LGTM!

**Pull request merged**
